docs/gen_test_case_reference.py

```diff
@@ -24,6 +25,19 @@
 
 logger = logging.getLogger("mkdocs")
 
+
+# if docs are generated while FAST_DOCS is true, then use "tests/frontier" otherwise use "tests"
+# USAGE 1 (use fast mode):
+#       export FAST_DOCS=true && uv run mkdocs serve
+# USAGE 2 (use fast mode + hide side-effect warnings):
+#       export FAST_DOCS=true && uv run mkdocs serve 2>&1 | sed '/is not found among documentation files/d' # noqa: E501
```

Comment:
@felix314159 I appended `# noqa: E501` to this comment to disable the line-too=long check; there's always an exception to the rule! In this case, it makes no sense to split the shell command over two lines and make it difficult for the user to copy-paste.